10_installation/11_install_agent/80_agent_configuration.txt

```diff
@@ -6,7 +6,7 @@ Configure the IP address or hostname of the Rudder Root Server in the following
 
 ----
 
-echo '<rudder server ip or hostname>' > /var/rudder/cfengine-community/policy_server.dat
+echo '<rudder server ip or hostname>' > /opt/rudder/etc/policy_server.dat
```

Comment:
i don't know if it's wise to change it now, since it's just a symlink, if it's not there this won't work.